src/test/java/uk/gov/pay/webhooks/queue/EventQueueIT.java

```diff
@@ -50,18 +50,19 @@ public void shouldReceiveMessageFromTheQueue() throws QueueException {
     
     @Test
     public void shouldConvertValidMessageFromQueueToEventMessage() throws QueueException {
-        
         var sqsMessage = """
                 {
-                	"serviceId": "my-id",
-                	"live": true,
-                	"timestamp": "2018-03-12T16:25:01.123456Z",
-                	"resource_external_id": "3uwuyr38rry",
-                	"eventType": "PAYMENT_CREATED",
-                	"event_details": {
-                		"example_event_details_field": "and its value"
-                	}
-                }""";
+                  "Type" : "Notification",
+                  "MessageId" : "04424f78-d540-5eb7-87e1-154d586b6b02",
+                  "Message" : "{\\\"sqs_message_id\\\":\\\"dc142884-1e4b-4e57-be93-111b692a4868\\\",\\\"service_id\\\":\\\"some-service-id\\\",\\\"live\\\":false,\\\"resource_type\\\":\\\"payment\\\",\\\"resource_external_id\\\":\\\"t8cj9v1lci7da7pbp99qg9olv3\\\",\\\"parent_resource_external_id\\\":null,\\\"event_date\\\":\\\"2019-08-31T14:18:46.446541Z\\\",\\\"event_type\\\":\\\"PAYMENT_DETAILS_ENTERED\\\",\\\"event_data\\\":{},\\\"reproject_domain_object\\\":false}",
```

Comment:
```suggestion
                  "Message" : "{\\"sqs_message_id\\":\\"dc142884-1e4b-4e57-be93-111b692a4868\\",\\"service_id\\":\\"some-service-id\\",\\"live\\":false,\\"resource_type\\":\\"payment\\",\\"resource_external_id\\":\\"t8cj9v1lci7da7pbp99qg9olv3\\",\\"parent_resource_external_id\\":null,\\"event_date\\":\\"2019-08-31T14:18:46.446541Z\\",\\"event_type\\":\\"PAYMENT_DETAILS_ENTERED\\",\\"event_data\\":{},\\"reproject_domain_object\\":false}",
```
IntelliJ says some of this escaping is redundant, so this should be equivalent.